bcipy/acquisition/protocols/lsl/lsl_connector.py

```diff
@@ -46,49 +49,87 @@ def inlet_name(inlet) -> str:
     return name.replace('-', '_')
 
 
-class LslDevice(Device):
-    """Driver for any device streaming data through the LabStreamingLayer lib.
+def channel_names(stream_info: pylsl.StreamInfo) -> List[str]:
```

Comment:
seems like these could be a part of the lsldevice class?

Reply:
I originally had this as a class method, but it didn't reference any instance variables and I got a linter warning about it so I pulled it out into a utility function. It may be useful in other contexts when we are looking at multiple LSL sources.